packages/patient-portal/src/views/DashboardView.tsx

```diff
@@ -14,7 +12,7 @@ import { MedicationsSection } from '../features/dashboard/MedicationSection';
 import { VaccinationsSection } from '../features/dashboard/Vaccinations/VaccinationsSection';
 
 export const DashboardView = () => {
-  const { data: patient, isLoading } = usePatientQuery();
+  const patient = useCurrentUser();
```

Comment:
I wonder if we should always refer to the logged in user as either "patient" or "user"?